testVMs.sh

```diff
@@ -0,0 +1,4 @@
+#just starts the vms and makes sure they're on the same network
+vagrant up
+ping 192.168.0.1
```

Comment:
This looks like it works, but can you add something to this script that checks if the hosts can talk to each other? Something like:
```
vagrant ssh box1 ping <box2 ip>
```